src/v/cluster/partition.h

```diff
@@ -184,6 +185,7 @@ class partition : public ss::enable_lw_shared_from_this<partition> {
     model::offset get_latest_configuration_offset() const;
 
     ss::shared_ptr<cluster::id_allocator_stm> id_allocator_stm() const;
+    ss::shared_ptr<cluster::jumbo_log_stm> jumbo_log_stm() const;
```

Comment:
why do we need JL stm in every partition?

Reply:
We don't need it. It's nullptr for all partitions except for kafka_internal.jumbo_log topic. This is copy & paste of the id_allocator_stm() accessor just above it. Need to re-review the code to see if it used. My guess it it's just a helper.

Reply:
done